File: 0035-search-insert-position/0035-search-insert-position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        // Iterate through list
        for (int i = 0; i < nums.length; i++){
            // Find index of target or index of first number greater than target
            if (nums[i] == target || nums[i] > target){
                return i;
            }
        }
        // If target is greater than any number in list, return length of list
        return nums.length;
    }
}

File: 0035-search-insert-position/0035-search-insert-position.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        # Iterate through list
        for i in range(len(nums)):
            # Find index of target or index of first number greater than target
            if nums[i] == target or nums[i] > target:
                return i
        # If target is greater than any number in list, return length of list
        return len(nums)
            